### PR TITLE
[cs9] Add rpm command to dump information about the packages in the container

### PR DESCRIPTION
**Pull request by aandvalenzuela**
`cs9` uses `fix_ssh_config.sh` from `shared` folder. `krb5.conf` and `ca-bundle.crt` are no longer used in the container, so I suggest removing them from the repository.

### CODE REVIEW

**Comment by aandvalenzuela** [comment]
It might be good to merge this PR before triggering a new rebuild of the image in https://github.com/cms-sw/cms-docker/issues/259

File: cs9/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,11 @@
 FROM @BASE_IMAGE_NAME@
-RUN dnf install -y \
-    automake bash bzip2 bzip2-libs bzip2-devel coreutils-single e2fsprogs e2fsprogs-libs \
-    perl file file-libs fontconfig freetype gcc-c++ git glibc krb5-libs libaio \
-    libcom_err libcom_err-devel libgomp libICE \
+
+ADD bin/rpm-pkg-info.sh /tmp/rpm-pkg-info.sh
+
+RUN dnf install -y bash && ./tmp/rpm-pkg-info.sh BASE &&\
```
Comment (smuzaffar) [inline cs9/Dockerfile:5]: @aandvalenzuela , why `./tmp/rpm-pkg-info.sh` (relative path) and not `/tmp/rpm-pkg-info.sh` ?

Comment (smuzaffar) [reply cs9/Dockerfile:5]: I think this is same for `alma8`

File: cs9/Dockerfile [hunk 2]
```diff
@@ -21,10 +24,11 @@ RUN dnf install -y \
...
 
-ADD fix_ssh_config.sh /tmp/fix_ssh_config.sh
+ADD share/fix_ssh_config.sh /tmp/fix_ssh_config.sh
 RUN mkdir -p /cvmfs /afs /eos /etc/vomses /etc/grid-security /build /data /pool /opt/cms &&\
     touch /etc/tnsnames.ora &&\
     echo el9 > /etc/cmsos &&\
     /tmp/fix_ssh_config.sh /etc/ssh/ssh_config && rm -f /tmp/fix_ssh_config.sh &&\
-    sed -i -e s'|^ *allow  *setuid.*|allow setuid = no|;s|^ *enable  *overlay.*|enable overlay = no|;s|^ *enable  *underlay.*|enable underlay = yes|' /etc/apptainer/apptainer.conf
+    sed -i -e s'|^ *allow  *setuid.*|allow setuid = no|;s|^ *enable  *overlay.*|enable overlay = no|;s|^ *enable  *underlay.*|enable underlay = yes|' /etc/apptainer/apptainer.conf &&\
+    ./tmp/rpm-pkg-info.sh IMAGE
```
Comment (smuzaffar) [inline cs9/Dockerfile:33]: same here

File: cs9/Dockerfile (hunk 1 above)
Comment (aandvalenzuela) [reply cs9/Dockerfile:5]: I thought both were fine since `cwd` is `/`, but let me change that

**Comment by smuzaffar** [comment]
+externals

**Comment by smuzaffar** [comment]
looks good, merging it

**Pull request merged**